apps/src/maze/collectorDrawer.js

```diff
@@ -0,0 +1,82 @@
+import Drawer from './drawer';
+const SQUARE_SIZE = 50;
+let SVG_NS = require('../constants').SVG_NS;
+let cellId = require('./mazeUtils').cellId;
+
+/**
+ * Extends Drawer to draw collectibles for Collector
+ */
+export default class CollectorDrawer extends Drawer {
+
+  /**
+   * @override
+   */
+  updateImageWithIndex_(prefix, row, col) {
+    let img = super.updateImageWithIndex_(prefix, row, col);
+    let val = this.map_.getValue(row, col);
```

Comment:
Nit: I'm not completely sure how these are consumed, but can some of the `let` definitions in this file be `const`?